Sources/XCTest/XCPredicateExpectation.swift

```diff
@@ -0,0 +1,58 @@
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//
+//  XCPredicateExpectation.swift
+//  Expectations with a specified predicate and object to evaluate.
+//
+
+#if os(Linux) || os(FreeBSD)
+    import Foundation
+#else
+    import SwiftFoundation
+#endif
+
+public class XCPredicateExpectation: XCTestExpectation {
```

Comment:
Actually, just one more thing - can we make this subclass have internal visibility, and return the base class type from the factory method on XCTestCase?